2015-12-22 11:07:58,355 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-22 11:08:31,387 INFO c.n.a.DeleteTables [main] all the tables were deleted
2015-12-22 11:13:29,528 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-22 23:51:33,034 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-22 23:53:06,846 INFO c.n.a.DeleteTables [main] all the tables were deleted
2015-12-22 23:53:28,344 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-23 11:44:24,134 INFO c.n.a.DeleteTables [main] all the tables were deleted
2015-12-23 11:44:35,997 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-23 21:30:05,708 INFO c.n.a.DeleteTables [main] all the tables were deleted
2015-12-23 21:30:11,880 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-23 22:28:00,663 ERROR c.n.d.i.AuthorDaoImpl [main] not able to get an author by Id
org.h2.jdbc.JdbcSQLException: No data is available [2000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:144) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.checkOnValidRow(JdbcResultSet.java:3195) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3201) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3218) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:339) ~[h2-1.4.190.jar:1.4.190]
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:55) [classes/:?]
	at com.nixsolutions.entity.QuickTest.main(QuickTest.java:10) [classes/:?]
2015-12-23 22:28:00,667 ERROR c.n.d.i.AuthorDaoImpl [main] throwing
com.nixsolutions.dao.DaoException: not able to get an author by Id
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:61) [classes/:?]
	at com.nixsolutions.entity.QuickTest.main(QuickTest.java:10) [classes/:?]
2015-12-23 22:28:00,671 ERROR c.n.d.i.AuthorDaoImpl [main] not able to get an author by Id
org.h2.jdbc.JdbcSQLException: No data is available [2000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:144) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.checkOnValidRow(JdbcResultSet.java:3195) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3201) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3218) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:339) ~[h2-1.4.190.jar:1.4.190]
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:55) [classes/:?]
	at com.nixsolutions.entity.QuickTest.main(QuickTest.java:33) [classes/:?]
2015-12-23 22:28:00,672 ERROR c.n.d.i.AuthorDaoImpl [main] throwing
com.nixsolutions.dao.DaoException: not able to get an author by Id
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:61) [classes/:?]
	at com.nixsolutions.entity.QuickTest.main(QuickTest.java:33) [classes/:?]
2015-12-24 10:51:04,761 INFO c.n.a.DeleteTables [main] all the tables were deleted
2015-12-24 10:51:09,237 INFO c.n.a.CreateTables [main] all the tables were created
2015-12-24 12:11:06,492 ERROR c.n.d.i.AuthorDaoImpl [main] not able to get an author by Id
org.h2.jdbc.JdbcSQLException: No data is available [2000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:144) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.checkOnValidRow(JdbcResultSet.java:3195) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3201) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3218) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:339) ~[h2-1.4.190.jar:1.4.190]
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:55) [classes/:?]
	at com.nixsolutions.AuthorDaoTest.testShouldretrieveAuthorById(AuthorDaoTest.java:30) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
	at junit.framework.TestCase.runTest(TestCase.java:176) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.runBare(TestCase.java:141) [junit-4.12.jar:4.12]
	at junit.framework.TestResult$1.protect(TestResult.java:122) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.runProtected(TestResult.java:142) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.run(TestResult.java:125) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.run(TestCase.java:129) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.runTest(TestSuite.java:252) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.run(TestSuite.java:247) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86) [junit-4.12.jar:4.12]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192) [.cp/:?]
2015-12-24 12:11:06,534 ERROR c.n.d.i.AuthorDaoImpl [main] throwing
com.nixsolutions.dao.DaoException: not able to get an author by Id
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:61) [classes/:?]
	at com.nixsolutions.AuthorDaoTest.testShouldretrieveAuthorById(AuthorDaoTest.java:30) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
	at junit.framework.TestCase.runTest(TestCase.java:176) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.runBare(TestCase.java:141) [junit-4.12.jar:4.12]
	at junit.framework.TestResult$1.protect(TestResult.java:122) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.runProtected(TestResult.java:142) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.run(TestResult.java:125) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.run(TestCase.java:129) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.runTest(TestSuite.java:252) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.run(TestSuite.java:247) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86) [junit-4.12.jar:4.12]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192) [.cp/:?]
2015-12-24 12:25:36,191 ERROR c.n.d.i.AuthorDaoImpl [main] not able to get an author by Id
org.h2.jdbc.JdbcSQLException: No data is available [2000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:179) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:155) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.message.DbException.get(DbException.java:144) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.checkOnValidRow(JdbcResultSet.java:3195) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3201) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3218) ~[h2-1.4.190.jar:1.4.190]
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:339) ~[h2-1.4.190.jar:1.4.190]
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:55) [classes/:?]
	at com.nixsolutions.AuthorDaoTest.testShouldretrieveAuthorById(AuthorDaoTest.java:30) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
	at junit.framework.TestCase.runTest(TestCase.java:176) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.runBare(TestCase.java:141) [junit-4.12.jar:4.12]
	at junit.framework.TestResult$1.protect(TestResult.java:122) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.runProtected(TestResult.java:142) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.run(TestResult.java:125) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.run(TestCase.java:129) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.runTest(TestSuite.java:252) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.run(TestSuite.java:247) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86) [junit-4.12.jar:4.12]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192) [.cp/:?]
2015-12-24 12:25:36,198 ERROR c.n.d.i.AuthorDaoImpl [main] throwing
com.nixsolutions.dao.DaoException: not able to get an author by Id
	at com.nixsolutions.dao.impl.AuthorDaoImpl.getAuthor(AuthorDaoImpl.java:61) [classes/:?]
	at com.nixsolutions.AuthorDaoTest.testShouldretrieveAuthorById(AuthorDaoTest.java:30) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
	at junit.framework.TestCase.runTest(TestCase.java:176) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.runBare(TestCase.java:141) [junit-4.12.jar:4.12]
	at junit.framework.TestResult$1.protect(TestResult.java:122) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.runProtected(TestResult.java:142) [junit-4.12.jar:4.12]
	at junit.framework.TestResult.run(TestResult.java:125) [junit-4.12.jar:4.12]
	at junit.framework.TestCase.run(TestCase.java:129) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.runTest(TestSuite.java:252) [junit-4.12.jar:4.12]
	at junit.framework.TestSuite.run(TestSuite.java:247) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86) [junit-4.12.jar:4.12]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382) [.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192) [.cp/:?]
